processes/security_team_membership_policy.md

```diff
@@ -0,0 +1,28 @@
+# Node.js Security Team Membership Policy
+
+The Node.js Security Team (@nodejs/security) has
+access to security-sensitive issues and patches that aren't approriate for
+public availability.
+
+> Note: Due to implementation details and a bit of human error, the list of
+> people with access to the security patches isn't 100% in line with the
+> @nodejs/security membership. See the [membership list](./security_team_members.md)
+> for details. This situation is intended to be corrected in the near future.
+
+This isn't to be confused with the security triage team, which is currently
+defined by the [`security@nodejs.org` email alias](https://github.com/nodejs/email/blob/master/iojs.org/aliases.json).
+Inclusion in that group is on a volunteer basis, upon approval by the Technical
+Steering Committee (ie. @nodejs/TSC).
+
+The policy for inclusion on the @nodejs/security is as follows:
+
+1. All members of the Technical Steering Committee (i.e. @nodejs/TSC) are
+   members of @nodejs/security.
+2. On a case-by-case basis, individuals outside the Technical Steering Committee
+   are invited by the TSC to join @nodejs/security so that their expertise can
+   be applied to an issue or patch. Once their assistance is no longer required
+   for a particular issue or patch, they are removed from the team.
```

Comment:
Giving people temporary access for one narrow issue means they have access to everything. I'm not opposed. Just pointing it out. I suspect folks on the security wg realized that and have already talked through it, but you know, hey, just in case....

Nit: If you're going to use `TSC`, then put `(TSC)` after the first mention (or at least some previous mention such as in line 21) of `Technical Steering Committee`.

Reply:
There has been talk of reducing the scope of access for "temporary" sec team members via the following: